lib/figaro.rb

```diff
@@ -0,0 +1,62 @@
+require 'yaml'
+require 'erb'
+
+class Figaro
+  def self.env
+    Environment
+  end
+
+  def self.setup(path)
+    Environment.setup(path)
+  end
+
+  def self.require_keys(keys)
+    Environment.require_keys(keys)
+  end
+
+  private
+end
+
+class Environment
+  def self.setup(path, env = Rails.env)
+    @config = {}
+    values = YAML.load(ERB.new(File.read(path)).result)
```

Comment:
AFAIK we don't actually put ERB stuff in `application.yml` .... can we get rid of that, for safety, simplicity etc? Also prefer `YAML.safe_load` if we can?

Reply:
👍🏼